itests/steps/autoscaler.py

```diff
@@ -223,6 +227,29 @@ def mesos_autoscaler(context):
 @behave.given('a kubernetes autoscaler object')
 def k8s_autoscaler(context):
     behave.use_fixture(autoscaler_patches, context)
+    context.mock_cluster_connector.return_value.__class__ = KubernetesClusterConnector
+    context.mock_cluster_connector.return_value.get_unschedulable_pods.return_value = (
```

Comment:
Same here, I think there should also be an unschedulable pod that has an unknown reason.